### DISCUSSION TITLE
vercel项目部署到github，还是给register.us.kg官方点了赞才封号的吗？

### DISCUSSION THREAD

**Discussion by ww936** [community/community#147736] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

What would be the problem with deploying a vercel project to github? Will it be blocked？
Give official kudos to register.us.kg, will it be blocked?

**Comment by ebndev** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @ww936! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
